bin/offline-dbg

```diff
@@ -133,6 +142,138 @@ EOF
     echo "**************************************************************"
 }
 
+get_highest_ofp_version() {
+    # if no protocol listed, default to OpenFlow14
+    hex_ofp_version=$(cat $cmd_dir/ovs-ofctl_-t_5_--version | grep "OpenFlow versions" | grep -oP [0-9]*x[0-9]* | tail -1)
+    version=${open_flow_protocols[$hex_ofp_version]}
+    if [ -z "$version" ]; then
+        echo "OpenFlow14"
+    else
+        echo $version
+    fi
+}
+
+do_collect-sos() {
+    OPTIND=1
+    while getopts "oh" opt; do
```

Comment:
what does the "o" do?
If only "-h" is needed, no need to add another option parsing